playspace_app/app/assets/javascripts/categories.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
if you guys are not going to use coffeescript, you should delete all those `.coffee` files